fix: Add connecting services to node + express (#3418)
getsentry/sentry-docs: src/docs/product/performance/trace-view.mdx

- [Ad blockers](/platforms/javascript/troubleshooting/#dealing-with-ad-blockers) may prevent transactions in browsers being sent, but HTTP requests to backend projects will still create child transactions
- [Rate-limiting](/product/accounts/quotas/#limiting-events) on a project may cause only some events to be sent to Sentry
- [Project permissions](/product/accounts/membership/#restricting-access) may mean you do not have access to transactions in another project

> - Exceeding the span limit, transactions are associated via the child spans of the parent transaction, but if there are more spans than the limit the association cannot be made

Reviewer: ```suggestion
- Exceeding the span limit. Transactions are associated via the child spans of the parent transaction, but if the number of spans exceed the limit, the association cannot be made
```
